adls-plugins/pom.xml

```diff
@@ -75,6 +75,14 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
why do we need this change

Reply:
This is to set the maven version to 1.8+

Reply:
Done